sections/dotnet.zsh

```diff
@@ -25,7 +25,7 @@ spaceship_dotnet() {
   [[ $SPACESHIP_DOTNET_SHOW == false ]] && return
 
   # Show DOTNET status only for folders containing project.json, global.json, .csproj, .xproj or .sln files
-  [[ -f project.json || -f global.json || -n *.csproj(#qN^/) || -n *.xproj(#qN^/) || -n *.sln(#qN^/) ]] || return
+  [[ -f project.json || -f global.json || -f paket.dependencies || -n *.csproj(#qN^/) || -n *.fsproj(#qN^/) || -n *.xproj(#qN^/) || -n *.sln(#qN^/) ]] || return
```

Comment:
This line is becoming too long. Let's split this up, like this:

```zsh
[[ -f project.json || -f global.json ||  -n *.csproj(#qN^/) || -n *.xproj(#qN^/) || -n *.sln(#qN^/) ]] || return
# F# project files
[[ -f paket.dependencies || -n *.fsproj(#qN^/) ]] || return
```

Reply:
Whoops. My bad this will not work the same way. You have to split the condition itself.

However, I think you've got the idea.

Reply:
Also maybe replace `-n *.csproj(#qN^/) || -n *.fsproj(#qN^/) || -n *.xproj(#qN^/)` with `-n *.(cs|fs|x)proj(#qN^/)`